release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2074,9 +2099,9 @@ This behavior has been fixed, and the references are now accepted and correctly
 [id="ocp-4-14-rhcos-bug-fixes"]
 ==== {op-system-first}
 
-* Previously, pods in {product-title} that access block PVC storage with LVM metadata could get stuck when terminating. This is because the same LVM devices were active both inside the container and on the host. An example of this occurred when running a virtual machine inside a pod using {VirtProductName} that uses LVM for the virtual machine. With this update, {op-system} by default only attempts to setup and access devices that are in the `/etc/lvm/devices/system.devices` file. This prevents contentious access to the LVM devices inside the virtual machine guests. (link:https://issues.redhat.com/browse/OCPBUGS-5223[*OCPBUGS-5223*])
+* Previously, pods in {product-title} that access block persistent volume claims (PVC) storage with logical volume manager metadata (LVM) could get stuck when terminating. This is because the same LVM devices were active both inside the container and on the host. An example of this occurred when running a virtual machine inside a pod that used {VirtProductName} that in turn used LVM for the virtual machine. With this update, {op-system} by default only attempts to setup and access devices that are in the `/etc/lvm/devices/system.devices` file. This prevents contentious access to the LVM devices inside the virtual machine guests. (link:https://issues.redhat.com/browse/OCPBUGS-5223[*OCPBUGS-5223*])
```

Comment:
```suggestion
* Previously, pods in {product-title} that access block persistent volume claims (PVC) storage with logical volume manager (LVM) metadata could get stuck when terminating. This is because the same LVM devices were active both inside the container and on the host. An example of this occurred when running a virtual machine inside a pod that used {VirtProductName} that in turn used LVM for the virtual machine. With this update, {op-system} by default only attempts to setup and access devices that are in the `/etc/lvm/devices/system.devices` file. This prevents contentious access to the LVM devices inside the virtual machine guests. (link:https://issues.redhat.com/browse/OCPBUGS-5223[*OCPBUGS-5223*])
```
"metadata" is not part of the storage-driver acronym